book/website/community-handbook/bookdash.md

```diff
@@ -87,6 +87,29 @@ It includes (but is not limited to) temporary access to high-speed internet, chi
 These bursaries are also available for the rent or purchase of small hardware items such as headphones or webcams to enhance participants' overall experience.
 * There is also financial support for subsistence costs such as buying meals or treats for the online social events during the Book Dash.
 
+## Organising a Local Hub
+As part of the wider online Book Dash event it is possible to organise local hubs that take place in person over 1-2 days in the same week as the online Book Dash. 
+In the past, these hubs have been organised at London and Bristol in the UK, as well as Amsterdam and Delft in the Netherlands.
+
+To organise a hub, you will need to consider which days are most suitable to organise the in-person event. 
+This may depend on your availability, the availability of rooms, or events that take place during the Book Dash event. 
+For the TU Delft Hub we always organised the in-person hub on the Tuesday and the Wednesday, so that we avoid the first online day which is focused on introductions, as well as the 'scriberia' day that usually takes place on Thursday. 
```

Comment:
```suggestion
For the TU Delft Hub we always organise the in-person hub on the Tuesday and the Wednesday, so that we avoid the first online day which is focused on introductions, as well as the 'scriberia' day that usually takes place on Thursday. 
```

I think this way it sounds less like the Delft hubs have finished.

Reply:
😄 I'll leave it up to you. Feel like it is nice to be aspirational about a Delft hub happening again.

Reply:
True - but hopefully it will be a Dutch hub and not just a TU Delft one :) 

Thanks for the review!